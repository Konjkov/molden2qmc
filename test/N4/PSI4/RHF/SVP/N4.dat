#!n4 high ang. mo. test
memory 1000 mb
molecule n4 {
symmetry = c1
0 1
N       0.5200      0.5300      0.5100
N      -0.5200     -0.5200      0.5200
N      -0.5200      0.5200     -0.5200
N       0.5200     -0.5200     -0.5200
}

set {
  basis def2-svp
  scf_type pk
  E_CONVERGENCE 10
  D_CONVERGENCE 10
  molden_write true
}

energy('scf')
