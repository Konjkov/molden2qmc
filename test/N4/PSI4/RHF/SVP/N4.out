    -----------------------------------------------------------------------
          PSI4: An Open-Source Ab Initio Electronic Structure Package
                              PSI 4.0 Driver

               Git: Rev {master} 521f06adb441ac1e2f1732eb1cd090190b409c49

    J. M. Turney, A. C. Simmonett, R. M. Parrish, E. G. Hohenstein,
    F. A. Evangelista, J. T. Fermann, B. J. Mintz, L. A. Burns, J. J. Wilke,
    M. L. Abrams, N. J. Russ, M. L. Leininger, C. L. Janssen, E. T. Seidl,
    W. D. Allen, H. F. Schaefer, R. A. King, E. F. Valeev, C. D. Sherrill,
    and T. D. Crawford, WIREs Comput. Mol. Sci. 2, 556-565 (2012)
    (doi: 10.1002/wcms.93)

                         Additional Contributions by
    A. E. DePrince, M. Saitow, U. Bozkaya, A. Yu. Sokolov
    -----------------------------------------------------------------------


    Psi4 started on: 05-13-2015  08:54:11 PM KRAT

    Process ID:   5361
    PSI4DATADIR: /home/kbob/git/psi4release/lib
    Memory level set to 256.000 MB

  ==> Input File <==

--------------------------------------------------------------------------
#!n4 high ang. mo. test
memory 1000 mb
molecule n4 {
symmetry = c1
0 1
N       0.5200      0.5300      0.5100
N      -0.5200     -0.5200      0.5200
N      -0.5200      0.5200     -0.5200
N       0.5200     -0.5200     -0.5200
}

set {
  basis def2-svp
  scf_type pk
  E_CONVERGENCE 10
  D_CONVERGENCE 10
  molden_write true
}

energy('scf')
--------------------------------------------------------------------------

  Memory set to 1000.000 MiB by Python script.

*** tstart() called on kbob-G41MT-S2
*** at Wed May 13 20:54:12 2015


         ---------------------------------------------------------
                                   SCF
            by Justin Turney, Rob Parrish, and Andy Simmonett
                              RHF Reference
                        1 Threads,   1000 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Full point group: C1

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z       
    ------------   -----------------  -----------------  -----------------
           N          0.520000000000     0.527500000000     0.512500000000
           N         -0.520000000000    -0.522500000000     0.522500000000
           N         -0.520000000000     0.517500000000    -0.517500000000
           N          0.520000000000    -0.522500000000    -0.517500000000

  Running in c1 symmetry.

  Rotational constants: A =      0.55979  B =      0.55648  C =      0.55324 [cm^-1]
  Rotational constants: A =  16782.11936  B =  16682.72627  C =  16585.64680 [MHz]
  Nuclear repulsion =  105.777940020337695

  Charge       = 0
  Multiplicity = 1
  Electrons    = 28
  Nalpha       = 14
  Nbeta        = 14

  ==> Algorithm <==

  SCF Algorithm Type is PK.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is CORE.
  Energy threshold   = 1.00e-10
  Density threshold  = 1.00e-10
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: DEF2-SVP
    Number of shells: 24
    Number of basis function: 56
    Number of Cartesian functions: 60
    Spherical Harmonics?: true
    Max angular momentum: 2

  ==> Pre-Iterations <==

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A         56      56       0       0       0       0
   -------------------------------------------------------
    Total      56      56      14      14      14       0
   -------------------------------------------------------

  Starting with a DF guess...

 OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
         stored in file 35.

  ==> Integral Setup <==

  ==> DFJK: Density-Fitted J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    OpenMP threads:              1
    Integrals threads:           1
    Memory (MB):               715
    Algorithm:                Core
    Integral Cache:           NONE
    Schwarz Cutoff:          1E-12
    Fitting Condition:       1E-12

   => Auxiliary Basis Set <=

  Basis Set: 
    Number of shells: 100
    Number of basis function: 308
    Number of Cartesian functions: 372
    Spherical Harmonics?: true
    Max angular momentum: 4

  Minimum eigenvalue in the overlap matrix is 7.3429304367E-03.
  Using Symmetric Orthogonalization.
  SCF Guess: Core (One-Electron) Hamiltonian.

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-RHF iter   1:  -198.88616886840217   -1.98886e+02   9.01315e-02 
   @DF-RHF iter   2:  -207.93848166880397   -9.05231e+00   7.65292e-02 DIIS
   @DF-RHF iter   3:  -217.09467561795978   -9.15619e+00   1.25289e-02 DIIS
   @DF-RHF iter   4:  -217.22974689642930   -1.35071e-01   7.73182e-03 DIIS
   @DF-RHF iter   5:  -217.30751470358572   -7.77678e-02   1.70214e-03 DIIS
   @DF-RHF iter   6:  -217.31336522180280   -5.85052e-03   3.12682e-04 DIIS
   @DF-RHF iter   7:  -217.31375793841400   -3.92717e-04   1.17854e-04 DIIS
   @DF-RHF iter   8:  -217.31380550841342   -4.75700e-05   6.75517e-05 DIIS
   @DF-RHF iter   9:  -217.31381483406008   -9.32565e-06   1.19571e-05 DIIS
   @DF-RHF iter  10:  -217.31381544465361   -6.10594e-07   2.62999e-06 DIIS
   @DF-RHF iter  11:  -217.31381548521901   -4.05654e-08   1.36756e-06 DIIS
   @DF-RHF iter  12:  -217.31381550141654   -1.61975e-08   4.75321e-07 DIIS
   @DF-RHF iter  13:  -217.31381550270058   -1.28404e-09   2.61997e-07 DIIS
   @DF-RHF iter  14:  -217.31381550291499   -2.14413e-10   8.79774e-08 DIIS
   @DF-RHF iter  15:  -217.31381550296854   -5.35465e-11   2.70936e-08 DIIS
   @DF-RHF iter  16:  -217.31381550297326   -4.71800e-12   3.28282e-09 DIIS
   @DF-RHF iter  17:  -217.31381550297320    5.68434e-14   7.65908e-10 DIIS
   @DF-RHF iter  18:  -217.31381550297326   -5.68434e-14   4.81843e-10 DIIS
   @DF-RHF iter  19:  -217.31381550297326    0.00000e+00   1.93129e-10 DIIS
   @DF-RHF iter  20:  -217.31381550297314    1.13687e-13   5.45883e-11 DIIS

  DF guess converged.

  ==> Integral Setup <==

 MINTS: Wrapper to libmints.
   by Justin Turney

   Calculation information:
      Number of atoms:                   4
      Number of AO shells:              24
      Number of SO shells:              24
      Number of primitives:             48
      Number of atomic orbitals:        60
      Number of basis functions:        56

      Number of irreps:                  1
      Integral cutoff                 0.00e+00
      Number of functions per irrep: [  56 ]

 OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
         stored in file 35.

      Computing two-electron integrals...done
      Computed 1244357 non-zero two-electron integrals.
        Stored in file 33.

	Batch   1 pq = [       0,    1596] index = [             0,1274406]
  ==> DiskJK: Disk-Based J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    Memory (MB):               715
    Schwarz Cutoff:          1E-12

   @RHF iter  21:  -217.31400798420910   -1.92481e-04   6.39883e-06 DIIS
   @RHF iter  22:  -217.31400803502405   -5.08149e-08   1.39084e-06 DIIS
   @RHF iter  23:  -217.31400803795358   -2.92954e-09   4.59481e-07 DIIS
   @RHF iter  24:  -217.31400803826355   -3.09967e-10   6.53037e-08 DIIS
   @RHF iter  25:  -217.31400803827731   -1.37561e-11   2.07228e-08 DIIS
   @RHF iter  26:  -217.31400803827896   -1.64846e-12   1.03576e-08 DIIS
   @RHF iter  27:  -217.31400803827941   -4.54747e-13   3.69150e-09 DIIS
   @RHF iter  28:  -217.31400803827924    1.70530e-13   4.09494e-10 DIIS
   @RHF iter  29:  -217.31400803827941   -1.70530e-13   1.31691e-10 DIIS
   @RHF iter  30:  -217.31400803827941    0.00000e+00   3.21074e-11 DIIS

  ==> Post-Iterations <==

    Orbital Energies (a.u.)
    -----------------------

    Doubly Occupied:                                                      

       1A    -15.752733     2A    -15.752071     3A    -15.752046  
       4A    -15.752028     5A     -1.667805     6A     -1.045909  
       7A     -1.042470     8A     -1.039081     9A     -0.694095  
      10A     -0.578091    11A     -0.576976    12A     -0.575797  
      13A     -0.549742    14A     -0.544431  

    Virtual:                                                              

      15A      0.184002    16A      0.186040    17A      0.187874  
      18A      0.282567    19A      0.285925    20A      0.289513  
      21A      0.740944    22A      0.870529    23A      0.871657  
      24A      0.872794    25A      0.924376    26A      0.924770  
      27A      0.925148    28A      0.928577    29A      0.930664  
      30A      1.071237    31A      1.071517    32A      1.071813  
      33A      1.106029    34A      1.234092    35A      1.237098  
      36A      1.240077    37A      1.888308    38A      1.890700  
      39A      1.892965    40A      2.125558    41A      2.129407  
      42A      2.383763    43A      2.549405    44A      2.552698  
      45A      2.555924    46A      2.651370    47A      2.657472  
      48A      2.662997    49A      2.900517    50A      2.910263  
      51A      3.340405    52A      3.343416    53A      3.346337  
      54A      3.545503    55A      3.551709    56A      3.557709  

    Final Occupation by Irrep:
              A 
    DOCC [    14 ]

  Energy converged.

  @RHF Final Energy:  -217.31400803827941

   => Energetics <=

    Nuclear Repulsion Energy =            105.7779400203376952
    One-Electron Energy =                -503.5373469116212277
    Two-Electron Energy =                 180.4453988530041499
    DFT Exchange-Correlation Energy =       0.0000000000000000
    Empirical Dispersion Energy =           0.0000000000000000
    PCM Polarization Energy =               0.0000000000000000
    EFP Energy =                            0.0000000000000000
    Total Energy =                       -217.3140080382794110

    Alert: EFP and PCM quantities not currently incorporated into SCF psivars.

Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr
  ==> Properties <==


Properties computed using the SCF density matrix

  Nuclear Dipole Moment: (a.u.)
     X:     0.0000      Y:    -0.0000      Z:    -0.0000

  Electronic Dipole Moment: (a.u.)
     X:    -0.0001      Y:     0.0033      Z:    -0.0033

  Dipole Moment: (a.u.)
     X:    -0.0001      Y:     0.0033      Z:    -0.0033     Total:     0.0047

  Dipole Moment: (Debye)
     X:    -0.0001      Y:     0.0083      Z:    -0.0084     Total:     0.0118


  Saving occupied orbitals to File 180.

*** tstop() called on kbob-G41MT-S2 at Wed May 13 20:54:15 2015
Module time:
	user time   =       2.88 seconds =       0.05 minutes
	system time =       1.25 seconds =       0.02 minutes
	total time  =          3 seconds =       0.05 minutes
Total time:
	user time   =       2.88 seconds =       0.05 minutes
	system time =       1.25 seconds =       0.02 minutes
	total time  =          3 seconds =       0.05 minutes

*** PSI4 exiting successfully. Buy a developer a beer!
